laos_gggi/ocean_heat_processing.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+import os
+
+
+def process_ocean_heat():
+    if not os.path.isfile("../data/ncei_global_ocean_heat.csv"):
+        url = "https://www.ncei.noaa.gov/data/oceans/woa/DATA_ANALYSIS/3M_HEAT_CONTENT/DATA/basin/onemonth/ohc_levitus_climdash_monthly.csv"
```

Comment:
Same as above -- take an `output_path` keyword argument in the function, and move this URL to the constants file